tests/test_console.py

```diff
@@ -4,13 +4,17 @@
 """
 
 import console
```

Comment:
This line is unnecessary after adding the line below it.

Reply:
This line is actually necessary to run the docstring tests previously written in the codebase.  The documentation is called by console.__doc__ so we have to import console to access its docstring.  The next line was added to more quickly reference HBNBCommand specifically as it is used more frequently.

Reply:
I had not seen the other uses of this in the file. Good catch!